### PR TITLE
Tweak What is Kontena section

### PR DESCRIPTION
**Pull request by nevalla**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -5,15 +5,16 @@ toc_order: 1
...
 
-With Kontena, you can construct your application using the **Kontena Service** definition. A Service definition describes the container image, networking, scaling and stateful/stateless attributes of your application. Services may be linked together to create the desired architecture. Each service is automatically assigned an internal DNS address, which can be used inside your application for inter-Service communications.
+With Kontena, you can construct your application using the **Kontena Service** definition. A Service definition describes the container images, networking, scaling and stateful/stateless attributes associated with your application. Services may be linked together to create the desired architecture. Each service is automatically assigned an internal DNS address, which can be used inside your application for communications between different Services.
```
Comment (jcolp) [inline docs/README.md]: This should be "communication" instead of "communications" I think.

Comment (nevalla) [inline docs/README.md]: Thanks for the hint. Changed it.

File: docs/README.md [hunk 2]
```diff
@@ -5,15 +5,16 @@ toc_order: 1
 
 # What is Kontena?
 
-[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster. The Kontena system is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
+[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster using multiple containers. The Kontena system is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
```
Comment (SpComb) [inline docs/README.md]: Do you mean `across a cluster of multiple machines` or `running applications consisting of multiple containers on a cluster`?

File: docs/README.md [hunk 3]
```diff
@@ -5,15 +5,16 @@ toc_order: 1
...
 
-With Kontena, you can construct your application using the **Kontena Service** definition. A Service definition describes the container image, networking, scaling and stateful/stateless attributes of your application. Services may be linked together to create the desired architecture. Each service is automatically assigned an internal DNS address, which can be used inside your application for inter-Service communications.
+With Kontena, you can construct your application using the **Kontena Service** definition. A Service definition describes the container images, networking, scaling and stateful/stateless attributes associated with your application. Services may be linked together to create the desired architecture. Each service is automatically assigned an internal DNS address, which can be used inside your application for communication between different Services.
```
Comment (SpComb) [inline docs/README.md]: The original wording is slightly better IMO

**Comment by jnummelin** [comment]
@nevalla  Still valid?

**Comment by SpComb** [comment]
Obsoleted by  #2451

**Pull request closed without merging**